include/orb_slam_2_ros/interface.hpp

```diff
@@ -43,6 +48,9 @@ class OrbSlam2Interface {
   // Helper functions
   void convertOrbSlamPoseToKindr(const cv::Mat& T_cv, Transformation* T_kindr);
 
+  virtual bool getBodyTransform()
```

Comment:
As above.

In addition please autoformat your code with clang format. If you use sublime, this is available here. A similar package exists for eclipse.

https://packagecontrol.io/packages/Clang%20Format